app/src/main/res/values/strings.xml

```diff
@@ -128,4 +128,9 @@
     <string name="title_bill_label">Sale amount</string>
     <string name="title_bill_description">Bill</string>
     <string name="title_total">Total</string>
+    <string name="msg_request_write_file">Write list device using</string>
+    <string name="title_update_photo"><![CDATA[Update Photo >>]]></string>
+    <string name="msg_error_enough_photos">Please make enough photos !!!</string>
+    <string name="msg_add_photo_sucessfully">You have added photos sucessfully !!!</string>
```

Comment:
WARNING: [Typos] "sucessfully" is a common misspelling; did you mean "successfully" ?
 This check looks through the string definitions, and if it finds any words that look like likely misspellings, they are flagged.